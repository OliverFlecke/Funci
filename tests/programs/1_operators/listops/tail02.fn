main = tail (True : False : True : [])