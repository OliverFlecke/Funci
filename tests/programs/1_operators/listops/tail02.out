Listy (Cons (Boolean False) (Cons (Boolean True) Empty))